stdlib/public/core/EmbeddedRuntime.swift

```diff
@@ -0,0 +1,121 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftShims
```

Comment:
Could this be an `@_implementationOnly` import of a private module, rather than `SwiftShims`? It's just that the latter gets installed (including all the headers) into `lib/swift` on all platforms, because things in it might be used from `@_alwaysEmitIntoClient` and/or `@inlineable` code.

Maybe you need `SwiftShims` for some other reason too — I'm just keen that we don't add extra headers to it if we don't have to.

Reply:
SwiftShims are needed throughout the entire stdlib for multiple reasons, so eliminating it here wouldn't achieve much.

Reply:
:-) I'm just keen that we don't put things in there that we don't _really_ need in `SwiftShims` per se. Not using it for things we don't really need it for would mean we could use a modulemap to pull in the appropriate C headers rather than redeclaring C library functions.

(I don't mind you not changing it here, FWIW.)